frontend/src/views/leaderboard/Leaderboard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import IRow from "./IRow";
+
+const Leaderboard: React.FC<{ rows: IRow[] }> = ({ rows }) => {
```

Comment:
Why are you using a closure instead of a standard function ?

Reply:
Also you need to add typing for the props. 
The convention is to declare an interface named `<ComponentName>Props`